### PR TITLE
BROKERS: Select All Shelters

### LINKED ISSUE #135: BROKERS: Select All Shelters

### PR DESCRIPTION
**Pull request by TylerDDev**
Closes #135

### CODE REVIEW

File: RefugeeLand.Core.Api/Brokers/Storages/StorageBroker.Shelters.cs [hunk 1]
```diff
@@ -26,5 +26,11 @@ public async ValueTask<Shelter> InsertShelterAsync(Shelter shelter)
 
             return shelterEntityEntry.Entity;
         }
```
Comment (Spectralgo) [inline RefugeeLand.Core.Api/Brokers/Storages/StorageBroker.Shelters.cs:28]: Hey @TylerDDev, Could you had an empty line between the two methods.

Comment (Spectralgo) [reply RefugeeLand.Core.Api/Brokers/Storages/StorageBroker.Shelters.cs:28]: Between the 28th and 29th line

Comment (TylerDDev) [reply RefugeeLand.Core.Api/Brokers/Storages/StorageBroker.Shelters.cs:28]: done

File: RefugeeLand.Core.Api/Brokers/Storages/IStorageBroker.Shelters.cs [hunk 2]
```diff
@@ -11,5 +11,6 @@ namespace RefugeeLand.Core.Api.Brokers.Storages
     public partial interface IStorageBroker
     {
         ValueTask<Shelter> InsertShelterAsync(Shelter shelter);
+        IQueryable<Shelter> SelectAllShelters();
```
Comment (Spectralgo) [inline RefugeeLand.Core.Api/Brokers/Storages/IStorageBroker.Shelters.cs:14]: @TylerDDev You are missing a using for LinQ on top of the file in order to use the IQueryable type.

**Review by Spectralgo — CHANGES_REQUESTED** [review]

**Comment by ElbekDeveloper** [comment]
LGTM

**Pull request merged**